<launch>  
    <arg name="drive_speed" default="1.0" />  
    <arg name="turn_speed" default="1.0" />  
    <arg name="joy_dev" default="/dev/input/js0" />  
    <arg name="cmd_topic" default="cmd_vel" />  
    
    <node pkg="joy" type="joy_node" name="joy_node">  
    <param name="dev" value="$(arg joy_dev)" />  
    <param name="deadzone" value="0.3" />  
    </node>  
    
    <node pkg="smartcar_teleop" type="teleop_joy.py" name="smartcar_teleop">  
    <param name="turn_scale" value="$(arg turn_speed)" />  
    <param name="drive_scale" value="$(arg drive_speed)" />  
    <remap from="cmd_vel" to="$(arg cmd_topic)" />  
    </node>  
</launch> 
